omero/developers/Model.txt

```diff
@@ -129,17 +129,16 @@ include:
 -  details
 -  … any SQL keyword
 
-
 Properties
-~~~~~~~~~~
-Mutable, annotated, global.
+^^^^^^^^^^
 
+Mutable, annotated, global.
```

Comment:
@joshmoore: Is this a to-do for expanding, or does it somehow continue the "not allowed" thread above?